spec/flipper/adapters/sequel_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'helper'
+require 'sequel'
+
+Sequel::Model.db =  Sequel.sqlite
```

Comment:
Does this create a file or is it in memory like I use with the active record adapter?